app/src/main/java/com/davidtomas/taskyapp/features/agenda/data/agenda/remote/mapper/AgendaMappers.kt

```diff
@@ -39,9 +40,14 @@ fun AttendeeResponse.toAttendeeModel() = AttendeeModel(
 
 fun PhotoResponse.toPhotoModel() = PhotoModel(
     key = key,
-    uri = url,
+    imageData = getBytesFromUrl(url)
 )
 
+private fun getBytesFromUrl(url: String): ByteArray {
+    val url = URL(url)
+    return url.readBytes()
+}
+
```

Comment:
It doesn't seem very efficient to me to load every image twice (once from Coil and once for yourself)